packages/cli/src/lib/webpack/render-html-plugin.js

```diff
@@ -45,7 +45,10 @@ module.exports = async function (config) {
 
 		// Unfortunately html-webpack-plugin expects a true file,
 		// so we'll create a temporary one.
-		const tmpDir = join(os.tmpdir(), 'preact-cli');
+		const tmpDir = join(
+			os.tmpdir(),
+			`preact-cli-${Math.floor(Math.random() * 100000)}`
+		);
```

Comment:
I think `tempy` is rather overkill for our needs, especially with the number of dependencies it adds: http://npm.anvaka.com/#/view/2d/tempy

A random number between 0 and 100,000 should be sufficient enough to avoid conflicts I think.